### ISSUE TITLE
User should be able to run nodule candidates prediction

### ISSUE THREAD

**Issue by Serhiy-Shekhovtsov** [concept-to-clinic/concept-to-clinic#285]
Currently we can only create a new case based on local images. There should be a button to start the actual prediction. After clicking it should show that prediction is running. Once prediction is ready - it should show update the status on the page.

## Checklist before submitting
- [ ] I have confirmed this using the officially supported Docker Compose setup using the `local.yml` configuration and ensured that I built the containers again and they reflect the most recent version of the project at the `HEAD` commit on the `master` branch 
- [ ] I have searched through the other currently open issues and am confident this is not a duplicate of an existing bug
- [ ] I provided a **minimal code snippet** or list of steps that reproduces the bug.
- [ ] I provided **screenshots** where appropriate
- [ ] I filled out all the relevant sections of this template

**Comment by Serhiy-Shekhovtsov** [comment]
Guys, if anyone is interested, feel free to take it.

**Comment by chaitanyarahalkar** [comment]
can anyone please explain to me how to use the prediction API? I am in a fix, trying to figure out how can I use it

**Comment by Serhiy-Shekhovtsov** [comment]
You can find the prediction code here: https://github.com/drivendataorg/concept-to-clinic/tree/master/prediction/src/algorithms
And here you can find some usage examples: https://github.com/drivendataorg/concept-to-clinic/tree/master/prediction/src/tests

**Comment by chaitanyarahalkar** [comment]
Hey, thanks for the help! I wanted to know whether it is possible to run the prediction code from the frontend on the sample LIDC images available in the Docker container by default.

**Comment by chaitanyarahalkar** [comment]
Has anyone worked on adding that button as @Serhiy-Shekhovtsov mentioned?
